### PR TITLE
Singular they

### PR DESCRIPTION
**Pull request by worldofpeace**
Resync of https://github.com/gopasspw/gopass/pull/1233 that catches new uses.

### CODE REVIEW

File: pkg/store/store.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 // RecipientCallback is a callback to verify the list of recipients
 type RecipientCallback func(context.Context, string, []string) ([]string, error)
 
-// ImportCallback is a callback to ask the user if he wants to import
+// ImportCallback is a callback to ask the user if they wants to import
 // a certain recipients public key into his keystore
```
Comment (Pharb) [inline pkg/store/store.go:15]:
```suggestion
// ImportCallback is a callback to ask the user if they want to import
// a certain recipients public key into their keystore
```

Comment (worldofpeace) [reply pkg/store/store.go:15]: oh, missed one

**Comment by martinhoefling** [comment]
@worldofpeace thx! Can you please signoff your commits to fulfill the DCO requirements?

**Comment by Pharb** [comment]
Sorry, I guess I messed the commit up with the Github Review suggestions feature 😞

**Comment by worldofpeace** [comment]
Did it.

**Comment by martinhoefling** [comment]
I think @Pharb , because you're co-author, you need to sign it off as well.

**Pull request merged**